MyToDo/App/Extensions/Date+.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  Date+.swift
+//  MyToDo
+//
+//  Created by Ivan Ruiz Monjo on 23/2/21.
+//
+
+import Foundation
+
+extension Date {
+    var addingWeek: Date {
```

Comment:
This probably works fine for your purposes right now but you might want to look into `DateComponents` and `Calendar` so you can literally bump a date by increasing its week component.

A possible issue with this code would be around weeks in which daylight savings start / end. You'd add 1 hour more or less than you want depending on the direction that the clock changes in.

Probably not relevant for what we're doing here but figured it's good to be aware of either way.

Reply:
Yes! Thanks for pointing it out. I know its not the best approax, but I want to move forwards and not fight with Dates.